WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -665,6 +666,10 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
     BlogDetailsSection *section = self.tableSections[sectionNum];
     BOOL hasTitle = section.title != nil && ![section.title isEmpty];
 
+    if ([[MySiteSettings alloc] isAssignedToExperiment] && sectionNum == 0 && (section.title == nil || section.title.isEmpty)) {
+        return FirstHeaderSectionHeight;
+    }
+
```

Comment:
Removing all this would ultimetely result in the same thing. If removed it will instead hit line 678 which returns the same value.

@leandroalonso If this makes sense to you, I can push this change.

Reply:
Just tested it and can confirm it has the same result. Plus it fixes height for headers with title, so it's a much better solution. You can go ahead and push it @hassaanelgarem, thank you!

Reply:
Pushed 409d282dd076dc139d52da1d435650ad7282ac02 👍